pkg/cluster/ipaddresses.go

```diff
@@ -52,14 +52,7 @@ func (m *manager) createOrUpdateRouterIPFromCluster(ctx context.Context) error {
 	// It might not exist if customer updated the profile or api is down.
 	// in both cases we can't do much so return early. Ingress profile can
```

Comment:
Can we remove or clean up this comment now that we are using a well named function I think it explains itself.